### PR TITLE
Add back CODEOWNERS

### PR DESCRIPTION
**Pull request by trozet**
Turns out we need CODEOWNERS to assign to the general team first, then auto assign will pick someone.

### CODE REVIEW

File: contrib/CODEOWNERS
Comment (tssurya) [reply contrib/CODEOWNERS:1]: also don't we have a cap on the github org membership settings? will we get bottlenecked when new people join/ we want to add more nvidia members there as well? (sorry for the silly questions I guess I should check who all are in this list first)

Comment (trozet) [reply contrib/CODEOWNERS:1]: we talked about this on slack. The branch protections are in place on master so that people cannot write to master. We can add additional branch protections to stop members from creating other branches in the repo.

**Review by tssurya — DISMISSED** [review]

**Review by tssurya — APPROVED** [review]

**Comment by tssurya** [comment]
will merge this when I get the "required statuses passes" thing green, LGTM

@trozet has communicated on slack with @jcaamano and @girishmg on the new process, we have consensus to try this out.

1. he has setup guardrails to ensure no one can edit master branch and merge things
2. given access to this list of 11 regular contributors to edit issues/ prs/ use the UI etc

**Pull request merged**
